schemas/specimen.json

```diff
@@ -331,10 +333,10 @@
         "displayName": "Percent Tumour Cells"
       },
       "restrictions": {
-         "range": {
-            "min": 0,
-            "max": 1
-          }
+        "range": {
+          "min": 0,
+          "max": 1
+        }
       }
```

Comment:
Pretty sure you meant to add the script here
```json
"script": "#/script/specimen/percentageTumourCells"
```

Reply:
Fairly sure you're right. The problem with rushing out the door lol

thanks!